extensions/chatgo/src/hooks/useQuestion.tsx

```diff
@@ -26,11 +29,11 @@ export function useQuestion(props: { initialQuestion: string; disableAutoLoad?:
             });
           }
         } catch (error) {
-          await showToast({
-            style: Toast.Style.Failure,
-            title: "Selected text couldn't load",
-            message: String(error),
-          });
+          // await showToast({
```

Comment:
why is this commented? Shouldn't it really show an error?
It works now on my side, but I'm worried that an error can occur and no feedback is given to the user.

Reply:
This error captures the error of getting the selected text API. It is reasonable to capture the error here but not display it on the UI

Reply:
That's alright, but I'd recommend removing the commented code and documenting why you're not treating the error.

Reply:
> That's alright, but I'd recommend removing the commented code and documenting why you're not treating the error.

ok~